bb_wrapper/services/barcode.py

```diff
@@ -59,7 +61,7 @@ def identify(self, number: str) -> Union[BarcodeCobranca, BarcodeTributo]:
         elif length == 44:
             data = {"barcode": number}
         else:
-            raise ValueError("Tipo não identificado!")
+            raise ValueError("Código de barras ou linha digitável inválida!")
```

Comment:
Se colocar um data vazio, vai passar e estourar as validações das duas instâncias? Caindo apenas no raise final? :thinking: 

```suggestion
            data = {}
```

Reply:
Sim, dessa forma ele caiu no raise final.